tests/integration/targets/test_mysql_user/tasks/issue-265.yml

```diff
@@ -79,65 +81,66 @@
         state: absent
       register: result
 
-    - name: assert changed is true for removing all blank users
+    - name: Issue-265 | Assert changed is true for removing all blank users
       assert:
         that:
           - result is not changed
 
-    - include: assert_no_user.yml user_name={{user_name_1}}
+    - include: utils_assert_no_user.yml user_name={{user_name_1}}
```

Comment:
Can we put these helper files in their own `utils` folder instead of using prefixes?